src/sidebar.jsx

```diff
@@ -127,11 +127,13 @@ export const SidebarPanelDetails = ({
     const [info, setInfo] = useState(null);
 
     useEffect(() => {
-        const filePath = path.join("/") + "/" + selected[0]?.name;
+        if (selected.length === 1) {
+            const filePath = path.join("/") + "/" + selected[0]?.name;
 
-        cockpit.spawn(["file", "--brief", filePath], { superuser: "try", error: "message" })
-                .then(res => setInfo(res?.trim()))
-                .catch(error => console.warn(`Failed to run file --brief on ${filePath}: ${error.toString()}`));
+            cockpit.spawn(["file", "--brief", filePath], { superuser: "try", error: "message" })
+                    .then(res => setInfo(res?.trim()))
+                    .catch(error => console.warn(`Failed to run file --brief on ${filePath}: ${error.toString()}`));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-233-20240131-113815-96d513f6-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)